.github/workflows/test-wheel-linux.yml

```diff
@@ -189,7 +189,7 @@ jobs:
           # We don't test compute-sanitizer on CTK<12 because backporting fixes is too much effort
           # We only test compute-sanitizer on python 3.12 arbitrarily; we don't need to use sanitizer on the entire matrix
           # Only local ctk installs have compute-sanitizer; there is not wheel for it
-          if [[ "${{ inputs.python-version }}" == "3.12" && "${{ inputs.cuda-version }}" != "11.8.0" && "${{ inputs.local-ctk }}" == 1 ]]; then
+          if [[ "${{ inputs.python-version }}" == "9.99" && "${{ inputs.cuda-version }}" != "11.8.0" && "${{ inputs.local-ctk }}" == 1 ]]; then
```

Comment:
The comment above this doesn't explain why we aren't running this anymore. Could we add a comment?

Reply:
This was/is just meant to be a stop-gap. I was hoping #593 or #594 might be ready in time.

Reply:
Can we just add a comment explaining why we're temporarily disabling it then? If #593 or #594 get blocked it would be good to have for the time being.

Reply:
I suppose this is resolved after merging #593.

Reply:
Yes! (I left a note on slack, not sure if you saw it?)
I'll resolve this here now.